Automattic/wp-calypso
# Jetpack Cloud: Update Backup VaultPress upsell to match current designs

#### Changes proposed in this Pull Request

Related to #44404.

* Alter the appearance of the Backup upsell page for VaultPress sites, to bring it up to date with current designs.

#### Testing instructions

* Select a VaultPress site in Calypso.
* Navigate to the **Jetpack > Backup** page.
* Verify that the page appears as it does in the below screenshots. Focus especially on the text copy, call-to-action button copy, and the VaultPress logo.

#### Screenshots

##### Desktop

<img width="1239" alt="Screen Shot 2020-07-23 at 10 57 07" src="https://user-images.githubusercontent.com/670067/88312880-54724a80-ccd8-11ea-8033-5158aa433685.png">

##### Tablet (logo should not appear)

<img width="782" alt="Screen Shot 2020-07-23 at 10 57 37" src="https://user-images.githubusercontent.com/670067/88312848-4c1a0f80-ccd8-11ea-9582-d49fe1c2f0d7.png">

##### Mobile

<img width="389" alt="Screen Shot 2020-07-23 at 10 57 49" src="https://user-images.githubusercontent.com/670067/88312833-46242e80-ccd8-11ea-9b1c-77408009b83b.png">
